plaid/processor_test.go

```diff
@@ -0,0 +1,57 @@
+package plaid
+
+import (
+	"strings"
+	"testing"
+
+	assert "github.com/stretchr/testify/require"
+)
+
+func TestCreateApexToken(t *testing.T) {
+	sandboxResp, _ := testClient.CreateSandboxPublicToken(sandboxInstitution, testProducts)
+	tokenResp, _ := testClient.ExchangePublicToken(sandboxResp.PublicToken)
+
+	// get test account
+	options := GetAccountsOptions{
+		AccountIDs: []string{},
+	}
+	accountsResp, err := testClient.GetAccountsWithOptions(tokenResp.AccessToken, options)
+	accountID := accountsResp.Accounts[0].AccountID
+
+	apexTokenResp, err := testClient.CreateApexToken(tokenResp.AccessToken, accountID)
+	assert.Nil(t, err)
+	assert.True(t, strings.HasPrefix(apexTokenResp.ProcessorToken, "processor-sandbox-"))
+
+}
+
+func TestCreateDwollaToken(t *testing.T) {
+	sandboxResp, _ := testClient.CreateSandboxPublicToken(sandboxInstitution, testProducts)
```

Comment:
shall we check the error ?

Reply:
we're already checking in plaid/sandbox_test.go — do we still need to check here?

Reply:
I think I meant something along the lines of:

"if the create sandbox public token request fails, then we are a going to get a null pointer error later in the test unless we check the error here"

does that make sense?

Reply:
makes sense ⚡️